package.json

```diff
@@ -100,5 +100,10 @@
     "*.css": [
       "stylelint"
     ]
+  },
+  "greenkeeper": {
+    "ignore": [
```

Comment:
Id prefer to keep this with the `greenkeeper.json` file too https://greenkeeper.io/docs#monorepo